CMakeLists.txt

```diff
@@ -11,22 +11,27 @@ include_directories(".")
 include_directories("exchange")
 include_directories("fxbattle")
 
-# sources
```

Comment:
y no glob?

Reply:
Glob is generally agreed as bad practice in Clark files.

And also... it didn’t work in the docket container for some reason

Reply:
Cmake*

Reply:
Bad practice -- probably, but listing all files can be annoying too :) Will have to live with it I guess